content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/count_products.rst

```diff
@@ -0,0 +1,96 @@
+=====================
+Inventory Adjustments
+=====================
+
+Inventory adjustments can be accessed through :menuselection:`Inventory --> Operations -->
+Inventory Adjustments`.
+The report has all products currently in the stock. Each line is a unique combination of
+- Location
+- Product
+- Lot/Serial
+- Package
+- Owner
+
+
+Add new product in your stock
+=============================
+
+To add a line, just hit *Create* and fill in product and counted quantity fields then press 
+`tab` or click *Save*
+
+.. image:: count_products/create-line.png
+   :align: center
+   :alt: apply all wizzard allows to change reason and accounting date
+
+At this stage, the count is recorded but not yet *applied*. Meaning the quantity on hand is 
+not yet updated. 
+
+Just press *Apply* on the line or button *Apply All* button on the top of the screen.
+
+*Apply All* allows to edit the reason to use for the adjustments applied
+
+
+.. image:: count_products/apply-all.png
+   :align: center
+   :alt: apply all wizzard allows to change reason 
+
+Count products
+==============
+
+Counting products is a recurent activity in a warehouse. It often happens that theoric 
```

Comment:
`Counting products is a recurent activity` to `Counting products is a recurring activity`